backend/corpora/common/entities/entity.py

```diff
@@ -62,3 +62,30 @@ def __getattr__(self, name):
         :return:
         """
         return self.db_object.__getattribute__(name)
+
+    @classmethod
+    def _create_sub_objects(
+        cls, rows: typing.List[dict], db_table: Base, keys: typing.List[str] = None, **kwargs
+    ) -> typing.List[Base]:
+        """
+        Create a list of Table Rows to be added to an Entity Object during an object creation. If id is provided, then
+        the row is retrieved and updated.
+
+        :param rows: A list of dictionaries containing columns.
+        :param db_table: The Table to add or modify rows
+        :param keys: additional primary keys
```

Comment:
Is this where keys are supplied to invoke "modify" over "insert" functionality? If so, is there an assumed order that must be maintained between `rows` and `keys`? Either way, looking for more clarification in documentation

Reply:
adding